tests/sentry/api/id_slug_path_params/test_doc_integration_slug.py

```diff
@@ -0,0 +1,26 @@
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from sentry.testutils.cases import BaseTestCase
+from sentry.testutils.helpers.options import override_options
+from sentry.testutils.silo import no_silo_test
+
+from .test_id_or_slug_path_params_mixin import APIIdOrSlugTestMixin
+
+
+@no_silo_test
```

Comment:
Is this temporary while we figure out hybrid cloud stuff? I you might also be able to remove it from all the resource test cases and just put it the main test class

Reply:
Are you referring to the import?

Reply:
oh no the `no_silo_test` decorator

Reply:
I am just going to keep it in the main test class, and yes, once we figure out hc, it should be removed